src/pages/Settings.vue

```diff
@@ -112,5 +134,16 @@ export default defineComponent({
   mounted() {
     this.contactRefreshInterval?.focus()
   },
+  watch: {
+    lang(lang) {
```

Comment:
You can do this in the `setup` function, but this is okay. I'm surprised this isn't already a ref.  There's a `watch()` function. This is preferred: https://vuejs.org/guide/essentials/watchers.html#basic-example

Reply:
I will look to improve that ; I'm not very familiar with vuejs